package.json

```diff
@@ -24,7 +24,8 @@
     "**/@walletconnect/ethereum-provider": "2.13.1",
     "**/@ethersproject/providers/ws": "7.5.10",
     "**/@synthetixio/synpress/ws": "8.17.1",
-    "**/elliptic": "6.6.0"
+    "**/elliptic": "6.6.0",
+    "cross-spawn": "7.0.5"
```

Comment:
don't need this line

delete it and run `yarn` again and you will see a small update on `yarn.lock` because there wasn't 7.0.7